tests/llmobs/_utils.py

```diff
@@ -832,6 +832,8 @@ def _expected_span_link(span_event, link_from, link_to):
 
 
 class TestLLMObsSpanWriter(LLMObsSpanWriter):
+    __test__ = False
```

Comment:
this is to prevent pytest warnings since the class name begins with `Test` and gets picked up as a test case class.